### ISSUE TITLE
Unable to run commands - 'Unauthorized'

### ISSUE THREAD

**Issue by propellor** [microsoft/powerbi-powershell#291]
Hi,
I am able to connect and get an access token.  However, I'm not able to run any other commands.  I have the Power BI Admin Role in m365.  

Getting this:

```
get-powerbidataset : Operation returned an invalid status code 'Unauthorized'
At line:1 char:1
+ get-powerbidataset -Scope Organization
+ ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
    + CategoryInfo          : WriteError: (Microsoft.Power...tPowerBIDataset:GetPowerBIDataset) [Get-PowerBIDataset], HttpO 
   perationException
    + FullyQualifiedErrorId : Operation returned an invalid status code 'Unauthorized',Microsoft.PowerBI.Commands.Data.GetP 
   owerBIDataset 
```

**Comment by CattieCat** [comment]
Are you using your user principal to connect using cmdlets? Are you able to access Power BI Admin Portal as tenant admin with the same credential? Example portal url:  https://app.powerbi.com/admin-portal/tenantSettings

**Comment by propellor** [comment]
Thanks for your help.  I run the connect command and then get my SSO login and select the account I admin with.

**Comment by CattieCat** [comment]
Let me rephrase my question. I'm trying to verify that you have delegated admin permissions. The easiest way to verify that is to see if you can use the same account to access powerbi admin portal. 
Are you able to run "Get-PowerBIDataset" without organization scope?

**Comment by propellor** [comment]
I am able to log into Power BI Admin Portal with this same account.  Running without org scope fails with the same error.  

"you have delegated admin permissions"  

Do you mean that I have performed an action to enable delegation?  Or do you mean I have the permissions, through delegation that my m365 admin performed? 

Not sure if I needed to do anything for my account beyond just being Power BI admin role in m365.

**Comment by propellor** [comment]
I had a co-worker try the same test and he got the same results.

**Comment by CattieCat** [comment]
This is quite weird, 
To scope down the issue, are you able to call PowerBI API directly? For both admin and non admin? https://docs.microsoft.com/en-us/rest/api/power-bi/

It'll also be helpful if you can share screenshot when login in with PowerShell and mask personal info.

**Comment by propellor** [comment]
Thanks for your suggestion, it was the key to solving it.  After finding the 'try it' functionality on a API page was generating an error "User is not licensed for Power BI", I spoke to one of our o365 admins and he said it was probably due to us running in UsGov environment.  I modified the Login command and was able to run subsequent commands. 

`Login-PowerBI -Environment USGov`

**Comment by jamesfgibson** [comment]
Excellent! I had the same issue and logging into the USGov Environment solved it.
Thanks!
